Scenes/BattleScenes/Battle.gd

```diff
@@ -1,15 +1,56 @@
 extends Node
```

Comment:
What was actually changed here, you moved a lot around

Reply:
I removed class_name Battle from the battle script because we didn't use it